### DISCUSSION TITLE
How to display a video using customtkinter?

### DISCUSSION THREAD

**Discussion by raiyan22** [TomSchimansky/CustomTkinter#1245] [General]
How to display a video inside the window that takes a certain portion of the window using customtkinter?

**Comment by Akascape** [comment]
@raiyan22 You can implement TkVideoPlayer with customtkinter:
An example is given here: https://github.com/TomSchimansky/CustomTkinter/discussions/531#discussioncomment-3922292

**Comment by IshanJ25** [comment]
you can break the videos into frames, then display back the frames using PIL image in a label
